src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/TrackExpensesAppControllerTest.java

```diff
@@ -0,0 +1,21 @@
+package pl.byczazagroda.trackexpensesappbackend.controller;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(MockitoExtension.class)
+class TrackExpensesAppControllerTest {
+
+    @InjectMocks
+    TrackExpensesAppController trackExpensesAppController;
+
+    @Test
+    void sayHello() {
+        String result = trackExpensesAppController.sayHello();
+        assertTrue(result.startsWith("TrackExpensesApp by"));
+    }
```

Comment:
There should be one empty line between 20 and 21.

Reply:
Done.